### ISSUE TITLE
Version2 beta v1 issues

### ISSUE THREAD

**Issue by dingquanyu** [KosinskiLab/AlphaPulldown#312]
- Need to remove the padded columns from the logits then recalculate iptm/iptm-ptm after the prediction
- The contents of the PAE plots are from the same model 
- Update AlphaLink2 backend to match with the workflow in run_structure_prediction 
- MonomericObject will crash here: https://github.com/KosinskiLab/AlphaPulldown/blob/05ae2d93ce94849327887f75cf7885408d893e60/alphapulldown/folding_backend/alphafold_backend.py#L279
